TaniaGayosso/HackerRank_August25/IfElse.java

```diff
@@ -0,0 +1,29 @@
+package challenges.one;
+
+import java.util.Scanner;
+
+public class IfElse {
+
+	private static final Scanner scanner = new Scanner(System.in);
+
+	public static void main(String[] args) {
+
+		final int NUMBER = scanner.nextInt();
+		final String MESSAGE;
+		scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+		if (NUMBER % 2 == 1) {
```

Comment:
I don't get it....
If NUMBER==3 then it's Weird... 
so why in line 18 you are evaluting: if (NUMBER >= 2 && NUMBER <= 5) ?? 
hmmmmmm.... something's wrong right?

Reply:
Yes, that's right. I will be fixing that.